check-diff.sh

```diff
@@ -395,7 +404,11 @@ function install_tools {
 			npm install -g jscs
 		fi
 
-		# @todo ESLint
+		# Install ESLint
+		if [ -n "$ESLINT_CONFIG" ] && [ -e "$ESLINT_CONFIG" ] && [ "$( type -t eslint )" == '' ] && ! grep -sqi 'eslint' <<< "$DEV_LIB_SKIP"; then
```

Comment:
Good point. Actually `-e` test for whether the file exists. I'll add that below as well.